ext/console/01_console.js

```diff
@@ -634,294 +651,282 @@ function formatRaw(ctx, value, recurseTimes, typedArray, proxyDetails) {
   ) {
     tag = "";
   }
-  let base = "";
-  let formatter = () => [];
-  let braces;
-  let noIterator = true;
-  let i = 0;
-  const filter = ctx.showHidden ? 0 : 2;
 
-  let extrasType = kObjectType;
-
-  if (proxyDetails !== null && ctx.showProxy) {
-    return `Proxy ` + formatValue(ctx, proxyDetails, recurseTimes);
-  } else {
```

Comment:
All this looks like a lot of changes, but I basically only hoisted out this `proxyDetails !== null` branch and moved it to a few lines above. So it's only indentation changes from here on.